client/ayon_substancedesigner/api/project_creation.py

```diff
@@ -62,6 +62,55 @@ def parse_graph_from_template(graph_name, project_template, template_filepath):
     return graph_element
 
 
+def parse_dependencies_from_template(template_filepath):
+    """Parse dependencies from Substance template file
+
+    Args:
+        template_filepath (str): Substance template filepath
+    """
+    dependencies = []
+    # Parse the template substance file
+    substance_tree = etree.parse(template_filepath)
+    substance_root = substance_tree.getroot()
+
+    for element in substance_root.find('.//dependencies'):
+        dependencies.append(element)
+    return dependencies
```

Comment:
Can we specify the return type for this function? I suppose it's `xml.etree.ElementTree.Element`?